test/e2e/pre_provisioning_test.go

```diff
@@ -174,7 +174,7 @@ var _ = ginkgo.Describe("Pre-Provisioned", func() {
 			skipIfUsingInTreeVolumePlugin()
```

Comment:
@andyzhangx This test case doesn't actually test shared disks in a meaningful way or that isn't already covered by other test cases. The previous test case handles creation of a disk with sharing capabilities, and the new test cases test mounting the shared disk to multiple nodes (which implies it can be mounted to one node). Also, this test case doesn't cover the supported scenario of sharing a block volume. I am inclined to simply remove it. Thoughts?

Reply:
sure, just go ahead remove it